src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -494,8 +505,59 @@ private void requestMaterials()
         final JobBuilder builderJob = (JobBuilder) job;
         while (builderJob.getStructure().findNextBlock())
```

Comment:
I'd say we should split this while-loop up into smaller segments... I'll bump up the continue/break limit once that is done